tests/torch_api/test_decentralized.py

```diff
@@ -78,6 +78,9 @@ def run_model(
     _init_bagua_env(rank, env)
     group = bagua.communication.new_group(ranks=list(range(nranks)))
 
+    if _rank_not_in_group(group):
+        return
```

Comment:
Can we keep the semantics of `with_bagua` not changed?

Reply:
_rank_not_in_group is rollback, it will be re-added in the later version.

Reply:
ok